app/javascript/components/shared/Error.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+function Error({ message }) {
+  return (
+    <div className="alert alert-warning" role="alert">
+      {message}
+    </div>
+  )
+}
+export default Error
```

Comment:
This was created with the idea of having multiple types of alerts. Probably it was supposed to be called Warning/Alert. However, I didn't need any other type, so I just left it like that for now.